pkg/utils/utils.go

```diff
@@ -558,35 +566,25 @@ func vfIsReady(pciAddr string) (netlink.Link, error) {
 	return vfLink, nil
 }
 
-func setVfsAdminMac(iface *sriovnetworkv1.InterfaceExt) error {
-	glog.Infof("setVfsAdminMac(): device %s", iface.PciAddress)
-	pfLink, err := netlink.LinkByName(iface.Name)
+func setVfsAdminMac(vfAddr string, pfLink netlink.Link, isRdma bool) error {
```

Comment:
imo the name: setVFAdminMac or setVFAdminMAC is more appropriate now.

Reply:
fixed